ghostTemplate.yaml

```diff
@@ -0,0 +1,377 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+Resources:
+  ghostFS:
+    Type: AWS::EFS::FileSystem
+    Properties:
+      PerformanceMode: generalPurpose
+      Encrypted: false
+  ghostMount0:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 0 
+        - !Ref DefaultVPCSubnets
+  ghostMount1:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 1 
+        - !Ref DefaultVPCSubnets
+  ghostMount2:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 2 
+        - !Ref DefaultVPCSubnets
+  ghostMount3:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 3 
+        - !Ref DefaultVPCSubnets
+  ghostAPI:
+    Type: AWS::ECS::TaskDefinition
+    Properties:
+      ContainerDefinitions:
+        - Image: registry.hub.docker.com/library/ghost
+          PortMappings:
+            - ContainerPort: 2368
+          LogConfiguration:
+            LogDriver: awslogs
+            Options:
+              awslogs-group: !Ref ghostAPILogGroup
+              awslogs-region: !Ref AWS::Region
+              awslogs-stream-prefix: logs
+          MountPoints:
+            - 
+              ContainerPath: /var/lib/ghost/content
```

Comment:
Usually we format things where the above two lines are collapsed into the same line. Just a cosmetic thing, though.